### PR TITLE
Updating CI files

### PR DESCRIPTION
**Pull request by VictorBjelkholm**
This commit updates all CI scripts to the latest version

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
     "star-signal": "node src/sig-server/bin.js",
     "lint": "aegir lint",
     "build": "aegir build",
-    "test": "aegir test --target node --target browser",
+    "test": "aegir test --target node --target browser --timeout 60000",
```
Comment (diasdavid) [inline package.json]: @VictorBjelkholm please don't. Increase the timeouts in tests.

Comment (VictorBjelkholm) [reply package.json]: Damn, thought "Longer timeout (for testing only)" was clear enough. This is just to check if there are timeout issues in CIs or if it's actual test failures.

Comment (diasdavid) [reply package.json]: Sorry, I didn't see the commit message, my bad.

**Comment by VictorBjelkholm** [comment]
Currently failing on windows, unsure if timeouts or real failures.


```
  42 passing (2m)
  6 pending
  11 failing

  1) signalling
       start and stop signalling server (default port) and spam it with invalid requests:
     Error: Timeout of 5000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves.
  

  2) signalling
       connect one client:
     Error: Timeout of 5000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves.
  

  3) signalling
       connect three more clients:
     Error: Timeout of 5000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves.
  

  4) signalling
       ss-join first client:

      Uncaught AssertionError: expected 0 to equal 1
      + expected - actual

      -0
      +1
      
      at Timeout.setTimeout [as _onTimeout] (test\sig-server.js:122:51)

  5) signalling
       ss-join and ss-leave second client:

      Uncaught AssertionError: expected 0 to equal 2
      + expected - actual

      -0
      +2
      
      at Timeout.setTimeout [as _onTimeout] (test\sig-server.js:130:51)

  6) signalling
       ss-join and disconnect third client:

      Uncaught AssertionError: expected 0 to equal 2
      + expected - actual

      -0
      +2
      
      at Timeout.setTimeout [as _onTimeout] (test\sig-server.js:142:51)

  7) signalling
       ss-join the fourth:
     Error: Timeout of 5000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves.
  

  8) signalling
       c1 handshake c4:
     Error: Timeout of 5000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves.
  

  9) signalling
       c1 handshake c2 fail (does not exist() anymore):
     Error: Timeout of 5000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves.
  

  10) signalling
       emits ws-peer every 10 seconds:
     Error: Timeout of 50000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves.
  

  11) transport: with wrtc
       dial
         dial on IPv4, check callback:
     Error: Timeout of 5000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves.
  



Command failed: mocha --timeout 5000 --ui bdd --colors --exit test\node.js test\**\*.spec.js
```

**Pull request merged**